frontend/src/components/modalContent/ViewIncidentModal.js

```diff
@@ -135,12 +135,25 @@ export class ViewIncidentModal extends Component {
           </div>
         )}
         <div className="inline-flex padding-small">
-          <p className="field-title">Who made the determination?</p>
+          <p className="field-title">Inspector who made the determination?</p>
```

Comment:
This isn't a question. E.g. Either "Which inspector made the determination?" or "Inspector who made the determination"

Reply:
I agree. I'll run it by UX and get these all updated